GermanFilter/sections/general_extensions.txt

```diff
@@ -59,6 +59,7 @@ dvdschnaeppchen.de#%#AG_onLoad(function() { window.myatu_bgm = 0; });
 chip.de#%#var originalUserAgent = navigator.userAgent; Object.defineProperty(navigator, 'userAgent', { get: function() { return originalUserAgent + ' Edge'; } });
 ! http://forum.adguard.com/showthread.php?5955
 menshealth.de#%#function google_ad_request_done() {};
+@@||ad1.adfarm1.adition.com/s$script,domain=derstandard.at

```

Comment:
wrong section. Is it anti-adblock rule?

Reply:
Yes for anti-adblock, when anti-adblock wall will start / show up, so after a few seconds of checking, it allows you to read another 5/6 articles:

> Die Deaktivierung war erfolgreich.
> Vielen Dank.
> <kbd>ZURÜCK ZUM INHALT</kbd>

It is possible the worst decision (advertising leakage / user privacy violation).

Reply:
@krystian3w https://forum.adguard.com/index.php?threads/policy-on-adblock-walls.26022/

Reply:
So this means it is not allowed to add this rule to any public filter list? If so, too bad!